python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -60,21 +60,16 @@
 import logging
 import numpy as np
 from .export_onnx import MXNetGraph as mx_op
-
-def import_onnx_modules():
-    """ To make sure ONNX is runtime dependency, it is imported used only when needed"""
-    try:
-        import onnx
-    except ImportError:
-        raise ImportError("Onnx and protobuf need to be installed. "
-                          + "Instructions to install - https://github.com/onnx/onnx")
-    return onnx
+try:
+    import onnx
+except ImportError:
+    onnx = None
```

Comment:
Any reason for not raising exception here?

Reply:
previously, onnx import was done in every conversion function as it is an optional dependency. moving it to a common import with ImportError exception will result in CI build failure. changing it to this way will ensure that there is no code duplication and CI build passes. since there are checks for onnx installation in other places and registration is not done when onnx is unavailable, this shouldn't cause an issue in normal scenarios.

Reply:
Got it. Thanks!